### PR TITLE
Updated OpenTelemetry.Extensions.Hosting to 1.*

### PR DESCRIPTION
**Pull request by RaulSebastian**
starting OpenTelemetry.Extensions.Hosting v1.4.0 the current OpenTelemetry logging sample doesn't work any more using *AddOpenTelemetryTracing(config => ...*

From the release notes:

> - Added `AddOpenTelemetry` extension ...
> - Removed deprecated extensions: `AddOpenTelemetryTracing` ...

Source: https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Extensions.Hosting/CHANGELOG.md#140

I updated the package version and the sample code to enable a quicker start and avoid having to search for the cause of the syntax error.

### CODE REVIEW

**Comment by cquirosj** [comment]
@RaulSebastian, thanks for sending this PR. We will be incorporating this change during our next maintenance window.

File: samples/open-telemetry/logging/Core_8/GenericHost/GenericHost.csproj [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <PackageReference Include="Microsoft.Extensions.Logging.EventLog" Version="6.*" />
     <PackageReference Include="NServiceBus" Version="8.*" />
     <PackageReference Include="NServiceBus.Extensions.Hosting" Version="2.*" />
-    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.0.0-rc9.4" />
+    <PackageReference Include="OpenTelemetry.Extensions.Hosting" Version="1.6" />
```
Comment (cquirosj) [inline samples/open-telemetry/logging/Core_8/GenericHost/GenericHost.csproj:11]: Consider where this should be 1.* instead of 1.6

Comment (RaulSebastian) [reply samples/open-telemetry/logging/Core_8/GenericHost/GenericHost.csproj]: thanks for the feedback, I've updated the version

**Comment by awright18** [comment]
@RaulSebastian Thanks for your contribution. Although this PR wasn't merged, the changes suggested here were incorporated with another [recent update](https://github.com/Particular/docs.particular.net/commit/2682777e01539ffc1c7647df168b896bf7b51a75) overhauling Open Telemetry samples.

**Pull request closed without merging**